src/main/java/it/gov/pagopa/payment/service/qrcode/expired/QRCodeExpirationBase.java

```diff
@@ -0,0 +1,35 @@
+package it.gov.pagopa.payment.service.qrcode.expired;
+
+import it.gov.pagopa.payment.model.TransactionInProgress;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public abstract class QRCodeExpirationBase {
```

Comment:
let's put "Base" as prefix, not suffix, in order to be able to discover and find it immediately